helpers/pdfUtils.py

```diff
@@ -10,11 +10,10 @@
 
 def get_pdf_for_url(url):
   try:
-    tmp = tempfile.NamedTemporaryFile(delete=True)
-    child = subprocess.Popen(["node", SCRIPT_PATH, "-w", tmp.name, "-u %s" % (url)], stdout=subprocess.PIPE)
-    child.communicate()
-    contents = tmp.read()
-    tmp.close()
-    return contents
+    with tempfile.NamedTemporaryFile(delete=True) as tmp:
+      child = subprocess.call(["node", SCRIPT_PATH, "-w", tmp.name, "-u %s" % (url)])
+      contents = tmp.read()
+      tmp.close()
```

Comment:
You don't need to call close, the `with` block handles that: http://effbot.org/zone/python-with-statement.htm

Reply:
right right, will remove